src/hb-subset.cc

```diff
@@ -67,7 +67,7 @@ template<typename TableType>
 static bool
 _subset2 (hb_subset_plan_t *plan)
 {
-  bool result = true;
+  bool result = false;
```

Comment:
Why this?

Reply:
this is to be consistent with _subset(), otherwise there's a unit test failure.